### DISCUSSION TITLE
add rotation view

### DISCUSSION THREAD

**Discussion by behzadrocket** [IENT/YUView#377] [Ideas]
One of the nice to have view modes is to be able to rotate display of a sequence by 90,180,270

**Comment by toaaot** [comment]
yes - rotating (and mirroring) the image would be very useful for me. Should be controllable by file name, too.

**Comment by ChristianFeldmann** [comment]
Ah but that is just for the rendering, right? But it may be a bit complicated with the other features. 
I files a feature request issue for this: https://github.com/IENT/YUView/issues/471
